src/pyramid_googleauth/__init__.py

```diff
@@ -0,0 +1,15 @@
+from pyramid_googleauth.routes import add_routes
```

Comment:
```suggestion
from pyramid_googleauth.routes import add_routes as _add_routes
```

I think we may want to clarify that `add_routes()` isn't part of the extension's public interface